RecoTauTag/RecoTau/python/deepTauSonicProducer_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+deepTauSonicProducer = cms.EDProducer("DeepTauIdSonicProducer",
+    Client = cms.PSet(
+        timeout = cms.untracked.uint32(300),
+        mode = cms.string("Async"),
+        modelName = cms.string("deepTau_2017v2p6_e6_full"),
+        modelConfigPath = cms.FileInPath("RecoTauTag/RecoTauTag-TrainingFiles/DeepTauId/deepTau_2017v2p6_e6_full/config.pbtxt"),
```

Comment:
Correct path to model config should be: `RecoTauTag/TrainingFiles/data/DeepTauId/deepTau_2017v2p6_e6_full/config.pbtxt`
at it is path where it will be stored when merged with cmsdist.
For local tests it should be copied as follows:
`git clone https://github.com/yongbinfeng/RecoTauTag-TrainingFiles/ RecoTauTag/TrainingFiles/data`